browser/js/w2ui-1.5.rc1.js

```diff
@@ -6197,6 +6197,10 @@ w2utils.event = {
                 }
             }
 
+            if (window.mode.isMobile()) { 
```

Comment:
Please remove the space after `{` it's appearing as a linting problem

Reply:
okay i will do that

Reply:
Done and pushed to the existing commit.